l1-contracts/deploy-scripts/DeployL1.s.sol

```diff
@@ -586,12 +586,15 @@ contract DeployL1Script is Script {
 
         Bridgehub bridgehub = Bridgehub(addresses.bridgehub.bridgehubProxy);
         bridgehub.transferOwnership(addresses.governance);
+        bridgehub.setPendingAdmin(addresses.chainAdmin);
 
         L1SharedBridge sharedBridge = L1SharedBridge(addresses.bridges.sharedBridgeProxy);
         sharedBridge.transferOwnership(addresses.governance);
+        sharedBridge.setPendingAdmin(addresses.chainAdmin);
 
         StateTransitionManager stm = StateTransitionManager(addresses.stateTransition.stateTransitionProxy);
         stm.transferOwnership(addresses.governance);
+        stm.setPendingAdmin(addresses.chainAdmin);
```

Comment:
should you also accept ownership?

Reply:
Since it's a script from one private key, i can't accept it right here i do it in the zk_toolbox

Reply:
this is the counterpart https://github.com/matter-labs/zksync-era/pull/2857